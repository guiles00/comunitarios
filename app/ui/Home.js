import React from "react";

const Home = function Home() {

  return (
    <div className="container-fluid pull-down ">  
      <div className="row justify-content-md-center">
        <div className="col-md-10">
          <fieldset>
            <div className="card">
              <div className="card-body justify-content-center">
                <h4 className="card-title text-center">
                 
                </h4>
                <hr></hr>
                <hr></hr>
                <h4 className="card-text text-center text-primary">
                  Holis... Bienvenido al sitio Comunitario&apos;s Jes&apos;s      
                </h4> 
              </div>
            </div>
          </fieldset>
        </div>
      </div>
    </div>
  );
};

export default Home;
