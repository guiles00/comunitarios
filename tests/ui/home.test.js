import Enzyme from 'enzyme';
import { shallow } from "enzyme";
import React  from "react";

import toJson from 'enzyme-to-json';

import Home from "../../app/ui/Home";

test("Should Match Snapshot",() =>{
  const wrapper = shallow(<Home />);
  expect(toJson(wrapper)).toMatchSnapshot();
});

test("Should Match Snapshot",() =>{
  const wrapper = shallow(<Home />);
  expect(wrapper.find("h4").length).toBe(2);
});