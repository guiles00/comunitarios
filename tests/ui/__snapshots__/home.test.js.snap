// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Match Snapshot 1`] = `
<div
  className="container-fluid pull-down "
>
  <div
    className="row justify-content-md-center"
  >
    <div
      className="col-md-10"
    >
      <fieldset>
        <div
          className="card"
        >
          <div
            className="card-body justify-content-center"
          >
            <h4
              className="card-title text-center"
            />
            <hr />
            <hr />
            <h4
              className="card-text text-center text-primary"
            >
              Holis... Bienvenido al sitio Comunitario's Jes's
            </h4>
          </div>
        </div>
      </fieldset>
    </div>
  </div>
</div>
`;
